test/index.js

```diff
@@ -126,21 +130,28 @@ const regulars = [
     params: 'a',
     args: ['a'],
     body: '',
-    defaults: {a: false},
+    defaults: {a: 'false'},
   },
   {
     name: null,
     params: 'a',
     args: ['a'],
     body: '',
-    defaults: {a: null},
+    defaults: {a: 'null'},
```

Comment:
Don't think those should be strings. I remember that i thought about that when built it. Wouldn't it be feel better to be actual values?